projects/demo/src/modules/app/app.routes.ts

```diff
@@ -1222,6 +1196,51 @@ export const ROUTES: Routes = [
             title: `Editor — HTML5`,
         },
     },
+    // ICONS
+    {
+        path: `icons/API`,
```

Comment:
```suggestion
        path: `icons/pack`,
```
?

See this:
* https://taiga-ui--pr4488-splincode-feat-icons-mfcciomp.web.app/icons/API/API